cnxdb/init/main.py

```diff
@@ -95,7 +103,13 @@ def _is_localhost_connection(db_connection):
 
 
 def init_venv(engine):
-    """Initialize a Python virtual environment for trigger importation."""
+    """Initialize a Python virtual environment for trigger importation.
+
```

Comment:
I know you just copied this part over, but how about:
Sets up database to use python virtualenv for python environment.

Afer this, plpythonu interpreters will execute within the virtual environment this code is executing in, rather than the default system python environment of the backend database. Note that this is only possible when executed locally on the backend server, or on a server with an identical path to a compatible python venv.

Reply:
Addressed in #99